docs/en/02_Developer_Guides/09_Security/00_Member.md

```diff
@@ -127,6 +127,13 @@ things, you should add appropriate `[api:Permission::checkMember()]` calls to th
 	  }
 	}
 
+## Saved user logins ##
```

Comment:
Capitalisation is a bit inconsistent, headline should be capitalised, and "Login", "Logs out" etc in content should be lower case.